packages/liveblocks-node/src/client.ts

```diff
@@ -929,6 +930,38 @@ export class Liveblocks {
     }
     return (await res.json()) as Promise<CommentData>;
   }
+
+  /**
+   * Creates a comment.
+   * 
+   * @param params.roomId The room ID to create the comment in.
+   * @param params.threadId The thread ID to create the comment in.
+   * @returns The created comment.
```

Comment:
**Nitpick:** Should we also include the comment in this JSDoc comment? even a very brief one.